packages/fields/src/Implementation.js

```diff
@@ -51,6 +57,12 @@ class Field {
     });
   }
 
+  // By default we assume that fields support unique.
+  // Fields should override this method if not.
+  get _supportsUnique() {
+    return true;
```

Comment:
I feel reasonably strongly that this isn't a safe default.

When creating a new field type, I shouldn't need to know to opt-out of behaviour that I'm not implementing.

This is more likely to be relevant for compound field types (e.g images, locations, embeds) etc than simple field types (text, numbers, etc) but given we have both, and in one case (creating a compound field) it's going to break by default, I'd prefer to flip the default

Reply:
Yep, I can get on board with this.

Reply:
Although that makes this a `major` breaking change.